service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/delete/DeleteAzureVmStep.java

```diff
@@ -0,0 +1,71 @@
+package bio.terra.workspace.service.resource.controlled.flight.delete;
```

Comment:
I didn't see a `DeleteAzureStorageAccountStep.java` (do we care about deleting storage accounts?)

Maybe that doesn't need to block this PR, but we should probably ticket it/add it at some point.

Reply:
I believe that @giventocode has that work on his todo list, but I don't have status on it. We should figure out where to track it.

Reply:
There are also gaps in the testing of storage account that are not addressed in this PR but should be done in a separate piece of work.